ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17-from-4.16.yaml

```diff
@@ -93,7 +80,7 @@ tests:
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
     workflow: openshift-upgrade-aws-heterogeneous
-- as: ocp-e2e-upgrade-azure-ovn-heterogeneous
+- as: ocp-e2e-upgrade-azure-ovn-multi-x-ax
```

Comment:
nit, lets say 'multi-x-xa' to be consistent with the power/z job names"multi-x-xp" and "multi-x-xz"

Reply:
Ah this was actually a typo on my part in the `x-xp` job name, then... My intension was to list them alphabetically.  If we want to list them in order of "introduction" to the cluster, that's fine, too, but more ambiguous.

Reply:
Actually, I was going for "alphabetized characters", and must have messed up the `multi-x-xp` job name... I'll fix that one to be consistent that way.